### PR TITLE
[Community spotlight] Simplifying license management and reducing costs with AWS and Xata

### LINKED ISSUE #261: [Illustration Request] Community Spotlight: Building a secure license management solution with Xata
Requesting an illustration please for the upcoming Community Spotlight due to publish 9/10 May.

Copy doc: https://docs.google.com/document/d/1k_RRS8Tg39HgfwVQRhO1fCaHMc7jeAWAaxo7ymc1JxI/edit#heading=h.y1e8chkl5dvr

Attached is the photo of Ilia.
IMG_20240508_092900

Also can we adapt the attached architecture image to include the Xata logo instead of the S3 bucket icon?

Screenshot 2024-05-08 093742

Thank you :)

### PR DESCRIPTION
**Pull request by CezzaineZ**
## Summary

- Draft: https://docs.google.com/document/d/1k_RRS8Tg39HgfwVQRhO1fCaHMc7jeAWAaxo7ymc1JxI/edit
- Recording: https://xata.rewatch.com/video/461wv6v3esg14qa0-ilia-gandelman-xata-community-spotlight
- Notes: https://www.notion.so/xata/Community-Spotlight-Ilia-Gandelman-April-5-2024-efc3e955b7ea43379c3861d2b981ae5b?pvs=4

Closes #261 

### Blog Release Checklist

- \[ ] Release on: \<05/13/24>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

**Review by CezzaineZ — COMMENTED** [review]
Looks great, thank you!

**Review by miukimiu — APPROVED** [review]
@CezzaineZ the text looks good. I've pushed the cover and architecture images.

LGTM! 🎉

**Comment by CezzaineZ** [comment]
Thanks @miukimiu - I have had a few copy amends from Ilia which I will add in later today for further review. But in the meantime, can you please kindly remove the text "s3 bucket" from the image? Not sure what else to replace it with... maybe "Database" or "Serverless database"? @alexfrancoeur any preference or thoughts here?

**Comment by miukimiu** [comment]
> But in the meantime, can you please kindly remove the text "s3 bucket" from the image? Not sure what else to replace it with... maybe "Database" or "Serverless database"

@CezzaineZ added **Serverless data platform** to be consistent with the website tagline. But let me know what's the final text decision once you and Alex have decided.

**Review by alexfrancoeur — APPROVED** [review]
Looks great!✅

As for the text -- serverless data platform works and is consistent with our messaging. The full line "Serverless data platform for PostgreSQL" or "Serverless PostgreSQL platform" would also work. I think it's important to be consistent though, so looks good as is to me.

File: community-spotlight-simplifying-license-management-aws-xata.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: 'Simplifying license management and reducing costs with AWS and Xata'
+description: 'Putting the spotlight on Ilia Gandelman, who significantly improved a cumbersome software license management system for smart rehabilitation therapy devices using Xata and AWS.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/community-spotlight-simplifying-license-management.jpg
+  alt: Community Spotlight
+author: Cezzaine Zaher
+date: 09-05-2024
```
Comment (alexfrancoeur) [inline community-spotlight-simplifying-license-management-aws-xata.mdx:8]:
```suggestion
date: 05-09-2024
```

@CezzaineZ I just noticed the date that we'll need to fix for sorting purposes. These dates are formatted in the US format actually. So it reads September 5th at the moment. I think we need to swap these.

File: community-spotlight-simplifying-license-management-aws-xata.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: 'Simplifying license management and reducing costs with AWS and Xata'
+description: 'Putting the spotlight on Ilia Gandelman, who significantly improved a cumbersome software license management system for smart rehabilitation therapy devices using Xata and AWS.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/community-spotlight-simplifying-license-management.jpg
+  alt: Community Spotlight
+author: Cezzaine Zaher
+date: 05-09-2024
```
Comment (CezzaineZ) [inline community-spotlight-simplifying-license-management-aws-xata.mdx:8]:
```suggestion
date: 05-13-2024
```

**Pull request merged**